scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/ModelValidatorContext.java

```diff
@@ -50,38 +51,46 @@
 
     private final Scheduler scheduler;
 
+    private final SchedulerSpaceInterface space;
```

Comment:
Thanks for this interesting review. Using Optional may make it more clear that space could be null, but since dealing with its null case is quite easy and straightforward, and after searching a bit more about Optional best practice ([ref1](https://stackoverflow.com/questions/23454952/uses-for-optional), [ref2](https://stackoverflow.com/questions/26327957/should-java-8-getters-return-optional-type)), I think it's not necessary to use Optional here.

Reply:
Ok, this is fine for me.